apps/judicial-system/web/messages/Core/titles.ts

```diff
@@ -251,11 +251,17 @@ export const titles = {
       defaultMessage: 'Öll mál - Réttarvörslugátt',
       description: 'Notaður sem titill fyrir öll mál skjá',
     }),
+    // TODO: Remove signedVerdictOverview
     signedVerdictOverview: defineMessage({
       id: 'judicial.system.core:titles.shared.signed_verdict_overview',
       defaultMessage: 'Yfirlit - Afgreitt mál - Réttarvörslugátt',
       description: 'Notaður sem titill fyrir Yfirlit úrskurðar skjá',
     }),
+    signedVerdictOverviewV2: defineMessage({
```

Comment:
Consider using the opportunity to change the name of the variable and the id in Contentful to something more meaningful that "signed verdict". Also consider changing the description.